File: member-service/src/main/java/com/hedvig/memberservice/commands/BankIdSignCommand.java
package com.hedvig.memberservice.commands;

import lombok.Value;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

import java.time.Instant;
import java.time.LocalDate;

@Value
public class BankIdSignCommand {
    @TargetAggregateIdentifier
    private Long id;

    private String referenceId;
    private String signature;
    private String oscpResponse;
    private Instant registeredOn;


}

File: member-service/src/main/java/com/hedvig/memberservice/commands/BankIdSignCommand.kt
package com.hedvig.memberservice.commands

import org.axonframework.commandhandling.TargetAggregateIdentifier

data class BankIdSignCommand(
    @TargetAggregateIdentifier val id: Long,
    val referenceId: String,
    val signature: String,
    val oscpResponse: String,
    val personalNumber: String
)
